events/views.py

```diff
@@ -371,8 +371,18 @@ def toggleJoinRequest(request, event_id):
     if not created:
         if join.status == PENDING:
             join.status = WITHDRAWN
+            # Create a notification when a request is withdrawn
+            Notification.objects.create(
+                user=event.creator,
+                message=f"{request.user.username} withdrew their request to join the event '{event.id}'.",
+            )
         else:
             join.status = PENDING
+            # Create a notification when a new request is submitted
+            Notification.objects.create(
+                user=event.creator,
+                message=f"{request.user.username} has requested to join the event '{event.id}'.",
+            )
         join.save()
```

Comment:
When I tested locally, it appears that when a user requests to join, the notification doesn't show a new one.
![1](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/950485f1-703a-4f31-bb3a-92bc81d7b64c)
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/54cd1d48-871b-41c6-a9e2-9cf217268a51)
Not sure if this is a local machine problem. Could you maybe take a look at it and make sure it's working on your side? Thank you!